estilos/algoritmo.css

```diff
@@ -73,3 +73,25 @@ h2 {
     margin-right: 5px;
 }
 
+#graphic-process {
+    margin: 30px 0;
+
+}
+
+.slot-ocupado, .slot-vacio {
+    border: 1px solid #999999;
+    float: left;
+    width: 25px;
+    padding: 1px;
+    text-align: center;
+}
+
+.slot-ocupado {
+    background: black;
```

Comment:
You are changing all the background properties, I guess you only want to change the `background-color` property.